.github/workflows/terraform.yml

```
@@ -25,7 +25,7 @@ jobs:
 
       # https://github.com/terraform-linters/setup-tflint
       - name: Install tflint
-        uses: terraform-linters/setup-tflint@v3.0.0
+        uses: terraform-linters/setup-tflint@v4.0.0
```

```suggestion
        uses: terraform-linters/setup-tflint@v4
```

Let's avoid redudant amounts of dependency bumps by referencing v4 which is available in this action and getting updated to be the latest v4